scripts/webpack/browser.js

```diff
@@ -74,7 +74,7 @@ module.exports = ({ isLegacyJS }) => ({
         rules: [
             {
                 test: /(\.tsx)|(\.js)|(\.ts)|(\.mjs)$/,
-                exclude: /node_modules\/(?!(@guardian\/discussion-rendering)|(@guardian\/types)|(dynamic-import-polyfill))\/.*/,
+                exclude: /node_modules\/(?!@guardian).*/,
```

Comment:
```suggestion
                exclude: /node_modules\/(?!@guardian\/).*/,
```
just on the off-chance we used something from `@guardian-random-other-scope`

Reply:
Good shout!